pkg/tsdb/cloudwatch/get_metric_query_batches_test.go

```diff
@@ -33,6 +34,24 @@ func TestGetMetricQueryBatches(t *testing.T) {
 		Expression:       "PERIOD(i1)",
 		Id:               "m1",
 	}
+	m99 := models.CloudWatchQuery{
```

Comment:
nit: we can change the naming convention for these to match the other test cases (eg. `m99_ref_m98` instead of `m99`)